### ISSUE TITLE
Pilot 3 data modeling

### ISSUE THREAD

**Issue by vafopoulos** [YourDataStories/ontology#24]
Provide updated properties and specifications for TED data based on Brussels plenary and technical meeting.

**Comment by niallob** [comment]
Final work being carried out today to verify sample data. 
Once this is complete a review to be carried out by tenforce.

**Comment by giorgosvaf** [comment]
About Uros following comment,

**Comment by giorgosvaf** [comment]
About Uros following comment on pilot 3 data model,

@Michalis: The eLOD master sheet says that the intended domain for elod:isRegisteredAt is organizations, not contracts. Can it be used in this case as well? (Actually, I would expect the range to be an institution, because of the "at" preposition.) 

We have to specify as Domain Class of elod:isRegisteredAt the Organization class because this property links an Agent with the country that owns the VAT number.

**Comment by niallob** [comment]
@Michalis It appears that this issue has been superseded by other issues. I would suggest that it be closed at this stage. (Issue #38 )
